web/src/app/schedule-view/component.html

```diff
@@ -7,6 +7,18 @@
       <span class="schedule-switch" (click) = "scheduleSet.incrementActiveSchedule()">&#9654;</span>
     </div>
     <div>
+      <div ngbDropdown class="d-inline-block">
+        <button class="schedule-button" id="dropdownBasic1" ngbDropdownToggle>
+          Sort schedules by...
+        </button>
+        <div ngbDropdownMenu>
```

Comment:
you can have this dropdown set an attribute on the component, that you can use when calling `Schedule.order`